test/libsolidity/syntaxTests/parsing/empty_tuple_assignment.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
+    function g() pure public returns (int, int, int) {}
+    function f() pure public {
+        // ensure this test is failing
+        (,,) = g();
+    }
+}
+// ----
+// ParserError: (150-151): Expected primary expression.
```

Comment:
I'm actually not sure this is what we want. If parsing is unambiguous, we could just allow it. This might be one way to get around an "unused return value" warning in the future.